plugin.go

```diff
@@ -2,15 +2,31 @@
 package web
 
 import (
+	"context"
+	"errors"
 	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/launchrctl/launchr/pkg/cli"
+	"github.com/launchrctl/launchr/pkg/log"
 
 	"github.com/launchrctl/launchr"
 	"github.com/launchrctl/web/server"
 	"github.com/spf13/cobra"
 )
 
-// APIPrefix is a default api prefix on the server.
-const APIPrefix = "/api"
+const (
+	// APIPrefix is a default api prefix on the server.
+	APIPrefix             = "/api"
+	stopArg               = "stop"
+	pluginName            = "web"
+	cobraBackgroundEnvVar = "BACKGROUND"
```

Comment:
Exclude cobra from naming. Let's try to not rely on the underlying technology. I plan to reduce direct cobra reference to core internal only.

Reply:
Did we agree on naming? Usually applications have a prefix not to overlap with global scope.  
But I still recommend to use bash for going background. Just add documentation how to use it, it's simple.
Why don't we use flags for that? Why we have foreground? But have background as env?